docs-site/content/docs/the-app/models.md

```diff
@@ -72,6 +72,11 @@ To add a new model _you have to use a migration_.
 $ cargo loco generate model posts title:string! content:text user:references
 ```
 
+When a model is added via migration, the following default values are provided:
```

Comment:
Thank you, @OdujokoD.

If the user includes fields such as `created_at`,`updated_at`,`create_at, or `update_at`, we are currently disregarding those specific fields. You can see the code [here](https://github.com/loco-rs/loco/blob/f0b4166f2e7678d50e4044e5b5ba3988987c0a23/src/gen/model.rs#L20).

Could you also document this behavior, please?